### DISCUSSION TITLE
Allow adding footnotes without a location parameter

### DISCUSSION THREAD

**Discussion by JohsEnevoldsen** [rstudio/gt#843] [Ideas]
It would be useful to add footnotes with `tab_footnote()` without specifying a location.
This should just add an entry to the footnote text without a superscript number.

My specific use case is to explain a symbol used in the table.

`tab_source()` can nearly do this, but it is displayed in a separate section

**Comment by rich-iannone** [comment]
This is a great idea! And definitely now is the time to do so (since footnotes and source notes were recently fixed up).

**Comment by rich-iannone** [comment]
Will create an issue from this. Thanks again.
